api/rs/slint/Cargo.toml

```diff
@@ -157,6 +157,9 @@ backend-linuxkms = ["i-slint-backend-selector/backend-linuxkms", "std"]
 ## windowing system. (Experimental)
 backend-linuxkms-noseat = ["i-slint-backend-selector/backend-linuxkms-noseat", "std"]
 
+## Use the backend based on the [android-activity](https://docs.rs/android-activity) crate. (Using it's native activity feature)
+backend-android-native-activity = ["i-slint-backend-android-activity/native-activity"]
```

Comment:
This is the name of the feature.
Other option include:
 - `backend-android-activity-native`  since it is using the native feature of the android-activity backend.
 - `backend-android-activity-0-5` of some variation of this to encode the version number of the android activity crate

Reply:
Would it perhaps make sense to synchronize a prefix of the feature name with the rust module?

Like we do for the software renderer and femtovg.

Like `backend-android-activity-native` and `slint::backends::android_activity::*`.

(This is a case of precedence, whatever we do here we can (and perhaps should) also do for other backends)

Reply:
Yes, i agree about the common prefix.
So I'll rename it `backend-android-activity-native`
Or should it be `backend-android-activity-native-0-5` or `backend-android-activity-native-05` or `backend-android-activity-05-native` ?